spec/fixtures/api/gpu-info.js

```diff
@@ -0,0 +1,17 @@
+const { app } = require('electron')
+
+app.commandLine.appendSwitch('--disable-software-rasterizer')
+
+app.on('ready', () => {
+  const infoType = process.argv.pop()
+  app.getGPUInfo(infoType).then(
+    (gpuInfo) => {
+      console.log(JSON.stringify(gpuInfo))
+      app.exit(0)
+    },
+    (err) => {
```

Comment:
> `err`

Can you call it `error` please? )

Reply:
`err` is error.

Reply:
`err` is not a word ) Can you please rename it?